test/conformance/tests/sriov_operator.go

```diff
@@ -1952,3 +1956,11 @@ func setSriovOperatorSpecFlag(flagName string, flagValue bool) {
 
 	}
 }
+
+// isSNO validates if the environment is a SNO (single node openshift) cluster
+// This is done by checking numer of nodes, it can later be substituted by an env variable if needed
+func isSnoCluster() bool {
```

Comment:
Can this go under the `cluster` package?
Like cluster.isSingleNode
I'd also remove references to OCP, as this is upstream

Reply:
Done